certified-connectors/TxtSync/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2235 @@
+                  },
+                  "TotalInboundSMS": {
+                    "type": "integer",
+                    "format": "int32",
+                    "title": "Total Inbound SMS",
+                    "description": "The total amount of inbound SMS recieved from this contact"
+                  },
+                  "TotalOutboundSMS": {
+                    "type": "integer",
+                    "format": "int32",
+                    "title": "Total Outbound SMS",
+                    "description": "The total amount of outbound SMS recieved to this contact"
+                  },
+                  "TotalFailedSMS": {
+                    "type": "integer",
+                    "format": "int32",
+                    "title": "Total Failed SMS",
+                    "description": "The total amount of outbound SMS fails to this contact"
+                  },
+                  "ExternalReference": {
+                    "type": "string",
+                    "title": "External Reference",
+                    "description": "External system reference for this contact"
+                  },
+                  "CompanyName": {
+                    "type": "string",
+                    "title": "Company Name",
+                    "description": "Comany name of the contact"
+                  },
+                  "AddressLine1": {
+                    "type": "string",
+                    "title": "Address Line1",
+                    "description": "Address line"
+                  },
+                  "AddressLine2": {
+                    "type": "string",
+                    "title": "Address Line2",
+                    "description": "Address line"
+                  },
+                  "City": {
+                    "type": "string",
+                    "title": "City",
+                    "description": "Address line"
+                  },
+                  "Postcode": {
+                    "type": "string",
+                    "title": "Postcode",
+                    "description": "Address line"
+                  },
+                  "County": {
+                    "type": "string",
+                    "title": "County",
+                    "description": "Address line"
+                  },
+                  "Country": {
+                    "type": "string",
+                    "title": "Country",
+                    "description": "Address line"
+                  },
+                  "LastCommunicationDate": {
+                    "type": "string",
+                    "description": "The date (UTC) that the contact last communicated",
+                    "title": "Last Communication Date",
+                    "format": "date-time"
+                  },
+                  "AllowSMS": {
+                    "type": "boolean",
+                    "description": "Whether the contact wishes to receive SMS",
+                    "title": "AllowSMS"
```

Comment:
@TxtSync There are few title values are looking like operation Ids. We expect title values must be free from grammatical and spelling mistakes.